### PR TITLE
Add skeleton for request show page in webui2

### PR DESCRIPTION
**Pull request by krauselukas**
The migration of the request view to bootstrap, requires
a basic skeleton

Co-authored-by: @bgeuken

### CODE REVIEW

**Comment by krauselukas** [comment]
[Request#Show Skeleton](https://trello.com/c/A3V1iUDN/1028-requestshow-skeleton)

**Comment by DavidKang** [comment]
@krauselukas, nice!, could you also add `switch_to_webui2` in the show action?

**Comment by krauselukas** [comment]
@DavidKang done, but i added a statement, so that it only switches to webui2 in development

**Comment by krauselukas** [comment]
@DavidKang i moved the switch_to_webui2 to the end of the show method, maybe you can check again :)

File: src/api/app/controllers/webui/request_controller.rb [hunk 1]
```diff
@@ -125,6 +125,9 @@ def show
 
     @comments = @bs_request.comments
     @comment = Comment.new
+
+    # TODO: Remove the statement after migration is finished
+    switch_to_webui2 if Rails.env.development?
```
Comment (dmarcoux) [inline src/api/app/controllers/webui/request_controller.rb]: Didn't we do that before (using `if Rails.env.development?`) and it caused specs to fail when we removed it just before finishing the deliverable?

I think the condition should be `if Rails.env.development? || Rails.env.test?`

**Pull request merged**